index.html

```diff
@@ -8,144 +8,227 @@
     <div>
       <!-- Landing -->
       <h1>
-        <img
-          src="file:///C:/Users/cmora/OneDrive/Pictures/Welcome.png"
-          width="600"
-        />
+        <div>
+          <img
+            src="file:///C:/Users/cmora/OneDrive/Pictures/Welcome.png"
```

Comment:
In order for this image to be able to render on a web server or someone else's computer (like mine while I'm testing your code), we'll need to move this image from where it is now on your harddrive into the folders that are part of this project, and commit them to the github repository. 

For example if you create an `/img` folder inside of this project, and move the image there, you could reference it like `src="./img/Welcome.png"`

If you check that in, then when we eventually host this project on the internet, or someone else downloads the repository, the image will work even when it's not on your own local machine!